src/test/ui/issues/issue-23281.rs

```diff
@@ -5,4 +5,8 @@ impl Struct {
     //~^ ERROR the size for values of type
 }
 
+struct Vec<T> {
+    t: T,
+}
```

Comment:
Another instance of a case where I think it would be worth duplicating the test.